docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -0,0 +1,22 @@
+# FAQ
+:label:`sec_faq`
+
+How can I perform inference on a file that won't fit in memory?
+
+The Tabular Dataset API works with pandas Dataframes, which supports chunking data in to sizes that fit in memory.
+Here's an example of one such chunk-based inference:
+
+```{.python .input}
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+predictor = task.fit(train_data=train_data)
```

Comment:
Limit row count of train data to speed up code. `train_data.sample(n=100, random_state=0)` and `task.fit(hyperparameters={'GBM': {}}, ...)`

Reply:
This is an invalid call to task.fit, need the label argument. `label='class'`